README.md

```diff
@@ -42,33 +52,48 @@ specification is located on
 This repository has all of its external c++ dependencies included in it. The
 excellent header-only JSON library
 [JSON for Modern C++](https://github.com/nlohmann/json) is included directly in
-the source code. The other two dependencies, the CSM API library, and gtest
-are included as git submodules. When you clone this library make sure you add
-the `--recursive` flag to your `git clone` command. Alaterntively, you can run
-`git submodule update --init --recursive` after cloning. The library can also be
-compiled against an installed versions of the CSM API by setting the BUILD_CSM
-flag to OFF during cmake configuration.
+the source code. The other three dependencies, The Abstraction Library for
+Ephemerides, the CSM API library, and googletest are included as git submodules.
+When you clone this library make sure you add the `--recursive` flag to your
+`git clone` command. Alterntively, you can run
+`git submodule update --init --recursive` after cloning.
+
+You can also install of the build requirements using Conda with the provided
+`environment.yml` file. The following commands will create a new environment
+to build against. Note that googletest cannot be installed via anaconda and must
+be available within the source code. You can remove the googletest dependency
+by disabling the tests.
 
-## Building CSM-CameraModel
+```
+conda env create -n usgscsm -f environment.yml -y
+```
 
-CSM-CameraModel uses a standard cmake build system. To compile the library, and
+## Building USGSCSM
+
+USGSCSM uses a standard cmake build system. To compile the library, and
 tests use the following commands:
 
 1. `mkdir build && cd build`
 2. `cmake .. && cmake --build .`
 
-## Testing CSM-CameraModel
+If you are using external dependencies via Conda or system level installations
```

Comment:
Thanks for adding these! The last time I built USGSCSM I ended up opening the Makefile to remind me what the flags were named.